common/src/main/java/com/wynntils/screens/OverlaySelectionScreen.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.mc.render.FontRenderer;
+import com.wynntils.mc.render.Texture;
+import com.wynntils.screens.lists.OverlayList;
+import com.wynntils.utils.objects.CommonColors;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.client.renderer.GameRenderer;
+import net.minecraft.network.chat.TranslatableComponent;
+
+public class OverlaySelectionScreen extends Screen {
+    private OverlayList overlayList;
+
+    public OverlaySelectionScreen() {
+        super(new TranslatableComponent("screens.wynntils.overlaySelection.name"));
+    }
+
+    @Override
+    protected void init() {
+        overlayList = new OverlayList(this);
+    }
+
+    @Override
+    public void render(PoseStack poseStack, int mouseX, int mouseY, float partialTick) {
+        poseStack.pushPose();
+
+        this.fillGradient(poseStack, 0, 0, this.width, this.height, -1072689136, -804253680);
```

Comment:
What are these magic numbers? If it's colors, can you make them constants and perhaps express them as RGB codes instead?

Reply:
I took this from the MC source code, I am not sure about those colors. It is used to render the gray background when a GUI is open. I will check this out.